### DISCUSSION TITLE
Welcome to OmegaDrive Discussions!

### DISCUSSION THREAD

**Discussion by Arcanthur** [Arcanthur/OmegaDrive#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by StrX77X** [comment]
Great theme !! just a couple questions.
can someone point me into the right direction as to 
- have roms displayed in TG16 instead of PCEngine
- remove NeoGeo games from the Arcade Classics Collection
Thanks
